base/abstractset.jl

```diff
@@ -117,14 +128,21 @@ julia> intersect([1, 2, 3], [3, 4, 5])
 1-element Vector{Int64}:
  3
 
-julia> intersect([1, 4, 4, 5, 6], [4, 6, 6, 7, 8])
+julia> intersect([1, 4, 4, 5, 6], [6, 4, 6, 7, 8])
 2-element Vector{Int64}:
  4
  6
 
-julia> intersect(Set([1, 2]), BitSet([2, 3]))
-Set{Int64} with 1 element:
-  2
+julia> intersect(1:16, 7:99)
+7:16
+
+julia> (0, 0.0) ∩ (-0.0, 0) 
+1-element Vector{Real}:
+ 0
```

Comment:
Whitespace bug:
```suggestion
julia> (0, 0.0) ∩ (-0.0, 0)
1-element Vector{Real}:
 0
```
Do we want these examples? They were trying to probe == vs isequal behaviour.